modules/nw-egress-ips-about.adoc

```diff
@@ -26,16 +26,28 @@ An egress IP address is implemented as an additional IP address on the primary n
 [id="nw-egress-ips-platform-support_{context}"]
 == Platform support
 
-While you can use the egress IP address functionality on any platform that {product-title} supports, the following platform specific considerations apply:
+Support for the egress IP address functionality on various platforms is summarized in the following table:
 
-Cloud platforms::
-Additional IP addresses must be assigned manually to appropriate nodes by an administrator.
+[IMPORTANT]
+====
+The egress IP address implementation is not compatible with Amazon Web Services (AWS), Azure Cloud, or any other public cloud platform incompatible with the automatic layer 2 network manipulation required by the egress IP feature.
+====
+
+[cols="1,1",options="header"]
+|===
+
+| Platform | Supported
 
-{rhosp}::
-Additional configuration by an administrator is necessary to allow {product-title} to automatically assign additional IP addresses to nodes.
+| bare metal | Yes
+| vSphere | Yes
+| {rhosp} | No
+| Public cloud^[1]^ | No
 
-vSphere::
-Additional configuration by an administrator is necessary to allow {product-title} to automatically assign additional IP addresses to nodes.
+|===
+[small]
+--
+1. Public cloud platforms, such as AWS and Google Cloud Platform (GCP), do not support the automatic layer 2 network manipulation required by the egress IP feature.
```

Comment:
Not sure if it matters, but you use different examples here than in the NOTE above the table. Also, is it necessary to have that same warning both places?

Reply:
However looking again, you already have "No" in the column for whether it's supported, so why would this note be necessary saying that these cloud platforms don't support the feature? The NOTE above made it sound like SOME public clouds might be supported if they have the "automatic layer 2 network manipulation required by the egress IP feature."

Reply:
@bergerhoffer, so while this feature might work with manual intervention for IP address assignment on some cloud providers, officially it was never tested by QE, so we don't actually support it on any cloud platform. And even if it did otherwise work, without automatic address assignment, it's incomplete.

There's probably no reason to include the footnote as well, though. Mostly it just looks awesome.